.circleci/config.yml

```diff
@@ -6,13 +6,61 @@ executors:
     environment:
       GIT_ORG: astronomer
     docker:
-      - image: circleci/python:3
+      - image: cimg/python:3.10
+
+orbs:
+  snyk: snyk/snyk@1.1.2
+  trivy: signavio/trivy@1.3.0
+
+parameters:
+  scan-docker-images:
+    type: boolean
+    default: false
+    description: "This will run docker image scan."
+
 jobs:
+  snyk-scan-docker:
+    executor: docker-executor
+    parameters:
+      docker_image:
+        type: string
+    steps:
+      - setup_remote_docker:
+          version: 20.10.12
+      - checkout
+      - run:
+          name: Pull Docker image
+          command: docker pull << parameters.docker_image >>
+      - snyk-docker-test:
+          docker_image: << parameters.docker_image >>
+  trivy-scan-docker:
+    executor: docker-executor
+    parameters:
+      docker_image:
+        type: string
+    steps:
+      - setup_remote_docker:
+          version: 20.10.12
+      - checkout
+      - run:
+          name: Pull Docker image
+          command: docker pull << parameters.docker_image >>
+      - trivy-docker-test:
+          docker_image: << parameters.docker_image >>
   run_pre_commit:
     docker:
       - image: quay.io/astronomer/ci-pre-commit:2022-04
     steps:
       - checkout
+      - run:
+          name: Installing required tools
+          command: |
+            apt update -y
+            apt install gnupg gawk apt-transport-https -y
+            curl https://baltocdn.com/helm/signing.asc | apt-key add -
+            echo "deb https://baltocdn.com/helm/stable/debian/ all main" | tee /etc/apt/sources.list.d/helm-stable-debian.list
+            apt-get update -y
+            apt-get install helm -y
```

Comment:
These steps should be moved into the ci-pre-commit so we're not doing this install every time, and failing CI jobs if the helm installation fails.

https://github.com/astronomer/ci-images/blob/main/helm-release/Dockerfile#L34

Reply:
This is being fixed in https://github.com/astronomer/ci-images/pull/34